appengine/app.go

```diff
@@ -11,12 +11,14 @@ import (
 const (
 	project = "GOOGLE_CLOUD_PROJECT"
 	bqTable = "GOOGLE_BIGQUERY_TABLE"
+	mode    = "BEACON_AUTH_MODE"
```

Comment:
Why BEACON_ ? All of this is to do with the beacon server...  I think "AUTHENTICATION_MODE" would be better